helper/common/common.go

```diff
@@ -174,3 +185,48 @@ func PadLeftOrTrim(bb []byte, size int) []byte {
 
 	return tmp
 }
+
+// FetchWhitelistFromConfig fetches whitelist object from the config
+// if doesn't exist returns empty map
+func FetchWhitelistFromConfig(genesisConfig *chain.Chain) map[string]interface{} {
```

Comment:
Nitpick. This is not genesis configuration

Reply:
Fixed ba983c6994b543fb581dfbd888c829a89c3000fe